ddocs/Makefile

```diff
@@ -0,0 +1,18 @@
+
+export PATH := ./node_modules/.bin:$(PATH)
+
+COUCH_URL ?= $$LOCAL_COUCH
+
+all: medic-api.json
+
+medic-api.json: medic-api/*.js
+	5984 compile medic-api/index.js > medic-api.json
+
+install: medic-api.json
+	# bug: does not fail on empty response
+	cat medic-api.json | \
+	  5984 fetch-revs "$(COUCH_URL)/medic" | \
+	  5984 bulk "$(COUCH_URL)/medic" | \
+	  tee /dev/stderr | \
+	  jq -e '.ok == true' > /dev/null
+
```

Comment:
Yeah I felt the same way and started on the grunt path and after mucking around with config files for an hour or something and gave up.  This is my personal standard way to build ddocs not medic's and I agree it should get updated to Grunt but wanted to move on to more important things.  I do plan to fix this before merge though.